### PR TITLE
Shiny R auth0 example

### PR DESCRIPTION
**Pull request by neelasha23**
----
📚 Documentation preview 📚: https://ploomber-doc--225.org.readthedocs.build/en/225/

### CODE REVIEW

**Comment by neelasha23** [comment]
Please review @edublancas

File: examples/shiny-r/app-with-auth0/app.R [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      p <- ggExtra::ggMarginal(p, type = margin_type, margins = "both",
+        size = 8, groupColour = input$by_species, groupFill = input$by_species)
+    }
+
+    p
+  }, res = 100)
+
+  output$welcome_message <- renderText({
+    auth_name <- read_header(session, "HTTP_X_AUTH_NAME")
+    auth_sub <- read_header(session, "HTTP_X_AUTH_SUB")
```
Comment (edublancas) [inline examples/shiny-r/app-with-auth0/app.R:68]: did you test this? this code looks the same as https://github.com/ploomber/doc/commit/76b55690a3567b4cc22e24b239c5fbc8c4687f8f but you added `HTTP_` to the header name, I'm guessing adding this prefix fixed the issue, how did you find the solution?

Comment (neelasha23) [reply examples/shiny-r/app-with-auth0/app.R:68]: yes I tested. I first printed all the available headers in the request and I saw that HTTP is prepended to all the properties.

Comment (neelasha23) [reply examples/shiny-r/app-with-auth0/app.R:68]: Screenshot 2024-05-17 at 10 16 24 AM

Logout also works:

**Review by edublancas — APPROVED** [review]

**Pull request merged**